src/lib/storage/table.cpp

```diff
@@ -9,6 +9,7 @@
 #include <thread>
 #include <utility>
 #include <vector>
+#include <stdexcept>
```

Comment:
Included files should be in lexicographical order. This would be assured by the format script.